mysql-test/suite/innodb/t/innodb_buffer_pool_resize_with_chunks.opt

```diff
@@ -1,3 +1,3 @@
 --innodb-buffer-pool-size=16M
---innodb-buffer-pool-chunk-size=2M
+--innodb-buffer-pool-chunk-size=4M
 --innodb-page-size=4k
```

Comment:
Why is this change necessary? It should be explained in the commit message if this file does not allow comments. I suppose that it is because of the reduced minimum buffer pool size and the change to the `.test` file.